proxy/src/registry.rs

```diff
@@ -247,6 +247,21 @@ impl Registry {
         Ok(self.client.get_user(user_id).await?.map(|_user| handle))
     }
 
+    pub async fn prepay_account(
```

Comment:
This is essentially a transfer, but now you can't specify the donator.

Reply:
This method is acting like a faucet, you can bootstrap a new account from the main available pool `Alice`. At this point there is no reason to have more flexibility until we actually wanna test transfers in the app. Also see this as temporary aid .